OKR/BIFROST.md

```diff
@@ -0,0 +1,69 @@
+# 🌈 IPFS Bifrost
+
+_The bridge to a content-addressed future._
+
+We support and improve the http-to-ipfs gateway. We run IPFS in production.
+
+## 2019 Q4 - Objectives and Key Results
+
+_What is the main thing?_
+
+- Provide actionable insights to IPFS developers
+- Provide a great service to the community
+
+**DRAFT** - the following are to be confirmed...
+
+- WIP: [bifrost 2019 Q4 planning document](https://hackmd.io/wHaeFVpLRRm-BK0z04FbZw)
+- TBD: [bifrost 2019 Q4 speedsheet](https://docs.google.com/spreadsheets/d/1VeyiLvBdX_PrP394kU_lwkQZxfNwqMVX1f7K4ursSPM/edit#gid=1439867466)
+
+### IPFS infrastructure informs and improves core ipfs development
+
+- **P0** -  Traffic is mirrored to staging gateway running latest go-ipfs.
+- **P0** -  All devs have access to ipfs logs from production and staging infrastructure.
+- **P1** -  Nightly and on-demand pprof profiles from all nodes are published to a shared repo.
+- **P2** -  Bifrost team identify 3 performance issues with go-ipfs.
+
+### IPFS infrastructure is stable and scales to meet demand
+
+- **P0** -  2 preload/delegate nodes are provisioned, monitored and documented
+  - companion is in brave now, so we need to ready for increased usage from js-ipfs
+- **P1** -  Ansible tower manages our deployments
+  - Running ansible from CI gets us so far, but will cause friction as preload/bootstrappers are brought into the bifrost-infra
+- **P1** -  4 new DNS Bootstrap nodes are provisioned, monitored, and documented.
```

Comment:
This could be a good opportunity to dogfood `/dns` multiaddrs (supporting both ip4 and ip6)
Ref. https://github.com/multiformats/go-multiaddr-dns/pull/17